cmake/tribits/core/package_arch/TribitsConstants.cmake

```diff
@@ -96,11 +96,11 @@ SET(${PROJECT_NAME}_PACKAGE_DEPS_FILES_DIR "cmake/dependencies")
 
 # Other stuff
 
-IF(WIN32)
+IF(${CMAKE_HOST_SYSTEM_NAME} STREQUAL "Windows")
```

Comment:
What happens on Cygwin?

Reply:
CMAKE_HOST_SYSTEM_NAME is CYGWIN on cygwin. Up to date versions of CMake do not define WIN32 on cygwin unless cross compiling using mingw (which is the correct behavior, because cygwin proper is POSIX not WIN32). As a result this change should not affect behavior on cygwin (and in particular, if cygwin git is installed it should work here). The one caveat there is that older versions of CMake do incorrectly define WIN32 on cygwin. I believe the behavior as implemented is correct, because building on cygwin, you do want to use the cygwin git build if at all possible to pick up any cygwin specific representations (e.g. symbolic links). It is possible that invoking the WIN32 git would work also, but it's be a bit of a Frankenstein thing to do.